pkg/server/config.go

```diff
@@ -42,6 +61,36 @@ func (cfg *HandlerConfig) Validate() error {
 	return nil
 }
 
+// ValidateMappingConfig validates the handler config for mapping service after load.
+func (cfg *HandlerConfig) ValidateMappingConfig() error {
```

Comment:
The `MappingHandlerConfig` should have its own `Validate` func and calls the `HandlerConfig.Validate`